tests/apps/core/test_templates_dns_prefetch.py

```diff
@@ -0,0 +1,45 @@
+"""
+Test for the dns prefetch
+"""
+from django.test.utils import override_settings
+
+from cms.test_utils.testcases import CMSTestCase
+
+from richie.apps.core.helpers import create_i18n_page
+
+
+class TemplatesDNSPrefetchTestCase(CMSTestCase):
+    """Testing the base.html template"""
+
+    @override_settings(MEDIA_URL="//xyz.cloudfront.net/media/")
+    def test_templates_media_cdn_dns_prefetch(self):
+        """
+        Check if using an CDN for the media, we are prefetching the DNS of its hostname.
+        """
+        homepage = create_i18n_page("my title", is_homepage=True)
+        homepage.publish("en")
+        url = homepage.get_absolute_url(language="en")
+        response = self.client.get(url)
+
+        self.assertEqual(response.status_code, 200)
+        self.assertContains(
+            response, r'<link rel="dns-prefetch" href="//xyz.cloudfront.net/media/">'
+        )
```

Comment:
I'll check that `preconnect` has not been append by default.
```suggestion
                self.assertNotContains(
            response,
            r'<link rel="preconnect" href="//xyz.cloudfront.net/media/" crossorigin>',
        )
```

Reply:
Right.